firefly/templates/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<title>{% if name %}{{ name }} | {% else %}{% endif %}firefly | API Browser</title>
+				<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+				<style type="text/css">
+						.bs-example{
+							margin: 20px;
+						}
+				</style>
+			</head>
+			<body>
+		<div class="container">
+		<br>
+		<h3>{% if name %}{{ name }} | {% else %}{% endif %}firefly | API Browser</h3>
+		{% if version %}
+		<p><b>Version: </b>{{ version }}</p>
+		{% endif %}
+		<div class="row">
+			<div class="col-md-6">
+				<div class="bs-firefly">
+					{% for function in functions %}
+					<div class="panel-group" id="accordion">
+						<div class="panel panel-default">
+							<div class="panel-heading">
+								<h4 class="panel-title">
+									<div class="pull-right">{{ function.path }}</div>
+									<a data-toggle="collapse" data-parent="#accordion" href="#collapse{{ loop.index }}">{{ function.name }}</a>
+								</h4>
+							</div>
+							<div id="collapse{{ loop.index }}" class="panel-collapse collapse in">
+								<div class="panel-body">
+									<p>
+										{% if function.doc %}
+											{{ function.doc }}
+										{% else %}
+											<i>No docstring provided</i>
+										{% endif %}
+									</p>
+									<h4>Parameters</h4>
+									<ul>
+										{% for param in function.parameters %}
+										<li>{{ param.name }}</li>
+										{% endfor %}
+									</ul>
+								</div>
+							</div>
+						</div>
+					</div>
+					{% endfor %}
+				</div>
+			</div>	
+			<div class="col-md-6">
+				<p>The API can be used by using firefly-client</p>
+				<p>Install it using:</p>
+				<div class="well">
+					<samp>
+					$ pip install firefly
+					</samp>
+				</div>
+				<p>Usage:</p>
+				<div class="well">
+					<samp>
+					>>> import firefly<br/>
+					>>> client = firefly.Client('{{ host_url }}')<br/>
+					>>> client.function_name(&lt;parameters&gt;)<br/>
+					&lt;function_response&gt;
+					</samp>
+				</div>				
+			</div>
+		</div>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+	</body>
+</html>
```

Comment:
Please split this into `layout.html` and `index.html`.

Reply:
Is that really needed? Future use cases?